bag_tools/scripts/check_drop.py

```diff
@@ -129,25 +134,25 @@ def check_drop(inbags, plot_format='png'):
 
       plt.subplot(311)
       plt.title(topic + ' - Message time difference [s]')
-      plt.plot(msg_time_diff_topic, 'b')
-      plt.plot(x, numpy.array([msg_time_diff_min, msg_time_diff_min]), 'r--')
-      plt.plot(x, numpy.array([msg_time_diff_max, msg_time_diff_max]), 'r--')
-      plt.plot(x, numpy.array([msg_time_diff_mean, msg_time_diff_mean]), 'g')
+      plt.plot(mt, msg_time_diff_topic,'b')
+      plt.plot(mt, msg_time_diff_min*numpy.ones(mt.shape), 'r--')
```

Comment:
This only draws an horizontal line, try using only the first and last point.
For me it's very slow if I create the `numpy.ones` vector all tme.